src/components/va-accordion/va-accordion.e2e.ts

```diff
@@ -79,4 +79,56 @@ describe('va-accordion', () => {
     expect(buttons[0].getAttribute('aria-expanded')).toEqual('true');
     expect(buttons[1].getAttribute('aria-expanded')).toEqual('true');
   });
+
+  it('fires an analytics event when expanded', async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
+      <va-accordion>
+        <va-accordion-item header="First item" subheader="First subheader">Some content</va-accordion-item>
+        <va-accordion-item header="Second item" subheader="Second subheader">A bit more</va-accordion-item>
+      </va-accordion>`,
+    );
+
+    const analyticsSpy = await page.spyOnEvent('component-library-analytics');
+
+    const buttons = await page.findAll(
+      'va-accordion-item >>> button[aria-expanded="false"]',
+    );
+    expect(buttons.length).toEqual(2);
+
+    await buttons[0].click();
```

Comment:
I was a bit confused by the expectation that there are two buttons, but we only click the first one. Can the second `<va-accordion-item>` be removed from `setContent` just to simplify things a bit? Maybe a `.find()` instead of `.findAll()` would help as well.

Reply:
Sounds good to me @bkjohnson...

https://github.com/department-of-veterans-affairs/component-library/blob/d612f7bd4732125834dc702f9edeb7fc12545071/src/components/va-accordion/va-accordion.e2e.ts#L83